kconfig-hardened-check.py

```diff
@@ -452,31 +465,34 @@ def check_config_file(fname):
                         help='check the config_file against these preferences')
     parser.add_argument('--debug', action='store_true',
                         help='enable internal debug mode')
+    parser.add_argument('--json', action='store_true',
+                        help='output results in json format')
     args = parser.parse_args()
 
     if args.debug:
         debug_mode = True
```

Comment:
The `debug_mode` is not compatible to json mode.
Let's add a check during the argument parsing.